src/test/cli.ts

```diff
@@ -19,20 +19,19 @@
 import { Command, Option } from 'commander';
 import fs from 'fs';
 import path from 'path';
-import type { Config } from './types';
-import { Runner, builtInReporters, BuiltInReporter } from './runner';
+import type { Config, ReporterDescription } from './types';
+import { Runner, builtInReporters } from './runner';
 import { stopProfiling, startProfiling } from './profiler';
 import { FilePatternFilter } from './util';
 import { Loader } from './loader';
 
 const defaultTimeout = 30000;
-const defaultReporter: BuiltInReporter = process.env.CI ? 'dot' : 'list';
 const tsConfig = 'playwright.config.ts';
 const jsConfig = 'playwright.config.js';
 const mjsConfig = 'playwright.config.mjs';
 const defaultConfig: Config = {
   preserveOutput: 'always',
-  reporter: [ [defaultReporter] ],
+  reporter: getDefaultReporters(),
```

Comment:
lets store this inside a variable, change the function return type to []string so you can use defaultReporters.join(", ") in the CLI command description.